spec/support/support_helper.rb

```diff
@@ -0,0 +1,19 @@
+module SupportHelper
+  def self.included(base)
+    included_ce(base)
+    included_pro(base) if defined?(Dradis::Pro)
+  end
+
+  def self.included_ce(base)
+    base.before(:each) do |example|
+      # Disable the setup tour
+      unless example.metadata[:skip_setup_mock]
```

Comment:
If the `skip_setup_mock` metadata doesn't exist, it mocks the `setup_required` callback. Shouldn't it be the other way around?